src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.html

```diff
@@ -6,105 +6,92 @@
 
-              <span class="kuiCheckBoxLabel__text">
-                Include system indices
-              </span>
-            </label>
+      <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--alignItemsEnd euiFlexGroup--responsive">
+        <div class="euiFlexItem euiFlexItem--flexGrowZero">
+          <div class="euiText">
+            <p class="euiTextColor euiTextColor--subdued">
+              Kibana uses index patterns to retrieve data from Elasticsearch indices for things like visualizations.
+            </p>
           </div>
         </div>
 
-        <p class="kuiText kuiSubduedText">
-          Kibana uses index patterns to retrieve data from Elasticsearch indices for things like visualizations.
-        </p>
+        <div class="euiFlexItem euiFlexItem--flexGrowZero">
+          <label
+            ng-if="controller.isSystemIndicesCheckBoxVisible()"
+            class="kuiCheckBoxLabel "
+            for="indexPatternCreationIncludeSystemIndices"
+          >
+            <input
+              type="checkbox"
+              id="indexPatternCreationIncludeSystemIndices"
+              class="kuiCheckBox"
+              ng-model="controller.doesIncludeSystemIndices"
+              ng-change="controller.onIncludeSystemIndicesChange()"
+            >
+
+            <span class="kuiCheckBoxLabel__text">
+              Include system indices
+            </span>
+          </label>
+        </div>
       </div>
 
-      <!-- User has no data -->
-      <div
-        class="kuiPanel kuiVerticalRhythm"
-        ng-if="!controller.hasIndices()"
-      >
-        <div class="kuiPanelBody">
-          <!-- Fetching state -->
-          <div
-            ng-if="controller.isFetchingExistingIndices"
-            class="kuiPanel kuiPanel--prompt kuiPanel--noBorder kuiVerticalRhythm"
-          >
-            <div class="kuiPanelBody">
-              <h2 class="kuiSubTitle kuiSubduedText kuiVerticalRhythm">
-                Checking for Elasticsearch data
-              </h2>
+      <div class="euiSpacer euiSpacer--m"></div>
 
-              <div class="kuiVerticalRhythm">
-                <p class="kuiText kuiSubduedText kuiVerticalRhythmSmall">
-                  <span class="kuiStatusText">
-                    <span
-                      aria-hidden="true"
-                      class="kuiStatusText__icon kuiIcon fa-spinner fa-spin"
-                    ></span>
-                    <span>
-                      Reticulating splines...
-                    </span>
-                  </span>
+      <div class="euiPanel euiPanel--paddingLarge">
+        <!-- User has no data -->
+        <div ng-if="!controller.hasIndices()">
+          <div class="euiPageContentBody">
+            <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentCenter euiFlexGroup--alignItemsCenter euiFlexGroup--responsive" style="height:100%">
```

Comment:
Can we replace this entire div with this:

```html
            <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--responsive">
              <!-- Fetching state -->
              <div
                ng-if="controller.isFetchingExistingIndices"
                class="euiFlexItem euiFlexItem--flexGrowZero createIndexPatternPrompt"
              >
                <h2 class="euiTitle euiTextColor euiTextColor--subdued">
                  Checking for Elasticsearch data
                </h2>
                <div class="euiSpacer euiSpacer--s"></div>
                <div class="euiFlexGroup euiFlexGroup--gutterSmall euiFlexGroup--alignItemsCenter">
                  <div class="euiFlexItem euiFlexItem--flexGrowZero">
                    <div
                      class="euiLoadingSpinner euiLoadingSpinner--medium"
                      aria-hidden="true"
                    ></div>
                  </div>
                  <div class="euiFlexItem euiFlexItem--flexGrowZero">
                    <p class="euiTextColor euiTextColor--subdued">
                      Reticulating splines...
                    </p>
                  </div>
                </div>
              </div>

              <!-- Empty state -->
              <div
                class="euiFlexItem euiFlexItem--flexGrowZero createIndexPatternPrompt"
                ng-if="!controller.isFetchingExistingIndices"
              >
                <div>
                  <h2 class="euiTitle euiTextColor euiTextColor--subdued">
                    Couldn't find any Elasticsearch data
                  </h2>
                  <div class="euiSpacer euiSpacer--s"></div>
                  <p class="euiTextColor euiTextColor--subdued">
                    <span>
                      You'll need to index some data into Elasticsearch before you can create an index pattern.
                    </span>
                    <a
                      class="kuiLink"
                      aria-label="Learn how to index data into Elasticsearch"
                      documentation-href="indexPatterns.loadingData"
                      target="_blank"
                      rel="noopener noreferrer"
                    >
                      Learn how.
                    </a>
                  </p>
                  <div class="euiSpacer euiSpacer--s"></div>
                  <button
                    class="euiButtonEmpty euiButtonEmpty--primary"
                    ng-click="controller.fetchExistingIndices()"
                  >
                    <span class="euiButton__content">
                      <span>
                        Check for new data
                      </span>
                    </span>
                  </button>
                </div>
              </div>
            </div>
```

This will:

* Uses a primary empty button to check for new data.
* Aligns text to the left so that the layout doesn't shift around so much when switching between the fetching and empty states.
* Shows a EUI spinner which the indices are being fetched.
* Adds another div inside of the loading state div to prevent the content from being laid out with flexbox rules. And then I also changed the style of the button to be empty and primary.

We also need to create `create_index_pattern_wizard.less` and add this class to make sure the empty and loading states don't get too wide:

```scss
.createIndexPatternPrompt {
  width: 100%;
  max-width: 600px;
}
```

Reply:
Nice, good idea!